README.md

```diff
@@ -68,6 +70,67 @@ Now, run the client:
 cargo run -- -c config.yaml  
 ```
 
+## Config reference
+```yaml
+log_level = "info"
+# Light client HTTP server host name (default: 127.0.0.1)
+http_server_host = "127.0.0.1"
+# Light client HTTP server port (default: 7000).
```

Comment:
Should we change default to be 7001?

Reply:
No need, I've changed the examples to port 7000 and 37000, for consistency.